docs/docsite/rst/dev_guide/developing_modules_best_practices.rst

```diff
@@ -135,7 +135,7 @@ Modules must output valid JSON only. Follow these guidelines for creating correc
 * Make returns reusable--most of the time you don't want to read it, but you do want to process it and re-purpose it.
 * Return diff if in diff mode. This is not required for all modules, as it won't make sense for certain ones, but please include it when applicable.
 * Enable your return values to be serialized as JSON with Python's standard `JSON encoder and decoder <https://docs.python.org/3/library/json.html>`_ library. Basic python types (strings, int, dicts, lists, and so on) are serializable.
-* Do not return an object via exit_json(). Instead, convert the fields you need from the object into the fields of a dictionary and return the dictionary.
+* Do not return an object trough exit_json(). Instead, convert the fields you need from the object into the fields of a dictionary and return the dictionary.
```

Comment:
+1 for getting rid of "via". In this instance I'd use "using" or "with" instead:
```suggestion
* Do not return an object using exit_json(). Instead, convert the fields you need from the object into the fields of a dictionary and return the dictionary.
```

Reply:
i also had feeling that there should've been another word:)